src/js/utils/log.ts

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable no-console */
-export const log: () => {} = (typeof console.log === 'function') ? console.log.bind(console) : noop;
+export const log: (msg?: string) => {} = (typeof console.log === 'function') ? console.log.bind(console) : noop;
```

Comment:
Oh! `(msg?: string) => {}` is just a type? Well then it should match `console.log`:

`(message?: any, ...optionalParams: any[]) => void`

Reply:
fixed